src/components/tx-flow/flows/TokenTransfer/index.tsx

```diff
@@ -29,8 +30,8 @@ type TokenTransferFlowProps = Partial<TokenTransferParams> & {
 
 const defaultParams: TokenTransferParams = {
   recipient: '',
-  tokenAddress: '',
-  amount: '',
+  tokenAddress: ZERO_ADDRESS,
```

Comment:
Won't this pre-select ETH? Do we want that?

Reply:
That's how I understood the design. It looks stange otherwise:

![image](https://github.com/safe-global/safe-wallet-web/assets/20442784/88a1f66c-9f1a-48f5-bf89-63988961d869)

Reply:
Perhaps it should be some placeholder text and a round skeleton if there's no native token? Like "⚪️ Token ▼"